### DISCUSSION TITLE
How to use databases.Database() with sqlalchemy.Table.create(engine)

### DISCUSSION THREAD

**Discussion by bobbetter** [encode/databases#400] [Q&A] [answered]
I have a sqlalchemy.Table object which has a create() method that would allow me to create the table in the database. It expects an 'engine' object. Is there a way I can use databases.Database() for this ? If i pass it directly i get:

`AttributeError: 'Database' object has no attribute '_run_ddl_visitor'`

Thank you.

**Comment by aminalaee** [comment, marked as answer]
So you want to create SQLAlchemy table?
There are some examples in `orm` repository which use `databases`.
Sorry I'm on phone, I will send an example later.

Update:

```python
import databases
import sqlalchemy

database = databases.Database("sqlite:///db.sqlite")
metadata = sqlalchemy.MetaData()


Table('my_table', metadata,
    Column('id', Integer, primary_key=True),
    Column('name', String(50))
)

engine = sqlalchemy.create_engine(str(database.url))
metadata.create_all(engine)
```

Something along these lines should work.

**Reply by bobbetter** [reply]
You mean this: https://github.com/encode/databases/blob/master/docs/database_queries.md ? 
I dont see how you would CREATE the table ? In this example the 'notes' table

**Reply by aminalaee** [reply]
```python
import databases
import sqlalchemy

database = databases.Database("sqlite:///db.sqlite")
metadata = sqlalchemy.MetaData()


Table('my_table', metadata,
    Column('id', Integer, primary_key=True),
    Column('name', String(50))
)

engine = sqlalchemy.create_engine(str(database.url))
metadata.create_all(engine)
```

Something along these lines should work.

**Reply by bobbetter** [reply]
thank you , i was thinking there was a way to avoid using 
`sqlalchemy.create_engine(str(database.url))` 
to create a "2nd engine" , i was thinking that databases.Database() already had an "engine" - but i am new to this. thanks again
